public/js/modules/module.js

```diff
@@ -3,17 +3,17 @@
 class Player {
     constructor(id,app,color,playerNum) {
         this.initialY = app.height/2
-        this.speed = 8
+        this.speed = 10
```

Comment:
Ever heard of magic numbers?

You're a true magician putting magic all over the code :D

Reply:
You´re right, I should create some constants for those values to be easy to track.